src/houghtransform.jl

```diff
@@ -86,3 +86,112 @@ function hough_transform_standard{T<:Union{Bool,Gray{Bool}}}(
     lines
 
 end
+
+"""
+```
+circle_centers, circle_radius = hough_circle_gradient(img, scale, min_dist, canny_thres, vote_thres, min_radius, max_radius)
+```
```

Comment:
Overall yes, but you should direct users to documentation about how they might obtain these arguments. See the use of `@ref` in various places in the Images, documentation, as well as ones that have an example in their docstring (e.g., the documentation for `imcorner` in https://github.com/JuliaImages/Images.jl/pull/617).